src/useSnapEdgeToParent/helpers.test.ts

```diff
@@ -93,28 +93,28 @@ test('Get layout returns right and bottom if element has crossed both edges', ()
   const breakDistance = 50
 
   // @ts-ignore
-  const layout1 = getPosition({ target, parent, container: { current: null } }, breakDistance)
-  expect(layout1).toMatchObject({ horizontal: 'right', vertical: 'bottom' })
+  const positioning1 = getPosition({ target, parent, container: { current: null } }, breakDistance)
+  expect(positioning1).toMatchObject({ horizontal: 'right', vertical: 'bottom' })
 
   // @ts-ignore
-  const layout2 = getPosition({ target, parent, container }, breakDistance)
-  expect(layout2).toMatchObject({ horizontal: 'right', vertical: 'bottom' })
+  const positioning2 = getPosition({ target, parent, container }, breakDistance)
+  expect(positioning2).toMatchObject({ horizontal: 'right', vertical: 'bottom' })
 })
 
-test('Get layout returns initial layout if one is provided', () => {
+test('Get positioning returns initial positioning if one is provided', () => {
   const target = { current: makeRect({ width: 400, height: 300 }) }
   const parent = { current: makeRect({ left: 400, top: 300 }) }
   const container = { current: makeRect({ right: 700, bottom: 500 }) }
 
   const breakDistance1 = 50
-  const initialLayout1 = { horizontal: 'right', vertical: 'bottom' }
+  const initialpositioning1 = { horizontal: 'right', vertical: 'bottom' }
```

Comment:
Typo here. Should be initialPositioning1. Happens multiple times in this file. Make sure you check if everything is ok after you change all occurrences of a string.

Reply:
done